### ISSUE TITLE
anyone still working on this?

### ISSUE THREAD

**Issue by AVVS** [chef/chef-provisioning-google#2]
subj, would appreciate the info

**Comment by tyler-ball** [comment]
Hey @AVVS - thanks for the interest.  Unfortunately the Chef provisioning team isn't currently working on this project, and its not in a useable state (even for non-production accounts).  It is still in a prototype phase.  I'll update if we continue working on it!

**Comment by mbrukman** [comment]
@temikus and @lykos are now actively working on this project.

Did you have any specific questions, @AVVS?

**Comment by kri5** [comment]
Is this developped still? It seems that there is no activity since a while...
Does Chef wants to support google compute engine?

**Comment by Temikus** [comment]
@kri5 Yep. @lykos has a PR in staging and I have some stuff in the works.
Do you have any specific requests/questions? 

PR's are also always welcome. If you have something - I'd be happy to review it.

Clarification: Me and @lykos are not employed by Chef inc. We're doing this as a side project as time permits.

**Comment by kri5** [comment]
Oh ok, nothing particular in mind, i'm using chef-provisionning-fog with gce driver as of now, but as i have seen yesterday chef announced a new version of aws cookbook, i was wondering if there were any efforts from chef support other cloud provider.

Keep the good work going on guys.

**Comment by Temikus** [comment]
@kri5 I may have limited visibility but there are definitely efforts from Chef side.
For example I know those projects are being contributed to actively by Chef:
https://github.com/chef/knife-google
https://github.com/test-kitchen/kitchen-google

@erjohnso may have a bit more details.
